### PR TITLE
upgrade readme

### PR DESCRIPTION
**Pull request by wuuuduu**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -93,15 +129,83 @@ and define some required methods:
...
+    FRONTEND_URL = "https://..."
+    BACKEND_URL = "https://..."
+
     GETPAID_ORDER_MODEL = 'yourapp.MyCustomOrder'
+    GETPAID_PAYMENT_MODEL = "yourapp.MyCustomPayment"
     GETPAID_PAYU_SLUG = "getpaid.backends.payu"
+    GETPAID_BACKEND_HOST = BACKEND_URL
+    GETPAID_FRONTEND_HOST = FRONTEND_URL
+
+    PAYMENT_URL = "{frontend_host}/payment/{payment_id}/"
```
Comment (szymborski) [inline README.rst]: Some kind of ```assert not FRONTEND_URL.endswith("/")``` may be useful to prevent incorrect configuration

File: README.rst [hunk 2]
```diff
@@ -110,21 +214,41 @@ Inform getpaid of your Order model in ``settings.py`` and provide settings for p
...
-    ]
+    class OrderViewSet(mixins.CreateModelMixin, viewsets.GenericViewSet):
+        serializer_class = OrderSerializer
+        queryset = Order.objects.all()
+        permission_classes = (permissions.IsAuthenticated,)
+
+        @transaction.atomic()
+        def perform_create(self, serializer):
+            super().perform_create(serializer)
+            self.create_payment(serializer.instance)
```
Comment (szymborski) [inline README.rst:234]: Could you please add some info where the payment_url is stored and when it's available to pass to a user?

File: README.rst (hunk 1 above)
Comment (szymborski) [reply README.rst]: Or leave a comment near url declarations about syntax

File: README.rst (hunk 2 above)
Comment (wuuuduu) [reply README.rst:234]: It was unused code - removed.

**Pull request merged**